go/vt/mysqlctl/mysqld.go

```diff
@@ -688,7 +687,12 @@ func deleteTopDir(dir string) (removalErr error) {
 // executeMysqlCommands executes some SQL commands,
 // using the mysql command line tool.
 func (mysqld *Mysqld) executeMysqlCommands(connParams *sqldb.ConnParams, sql string) error {
-	return mysqld.executeMysqlScript(connParams, strings.NewReader(sql))
+	params, err := dbconfigs.WithCredentials(connParams)
```

Comment:
I know the no-password root user is depended on elsewhere as well, so it's fair if we want to punt on that in this PR. I was planning on digging into this internally, and seeing if it made sense to PR it up ourselves or if we could add the no-password root user back in.